baselines/dom.generated.d.ts

```diff
@@ -3796,7 +3796,12 @@ declare var FocusNavigationEvent: {
 }
 
 interface FormData {
-    append(name: any, value: any, blobName?: string): void;
+    append(name: string, value: string | blob, fileName?: string): void;
```

Comment:
You're totally right, thanks so much for catching that. I just fixed it and it seems to be passing now. Thank you!